javascript/javascript1/week2/student_manager.js

```diff
@@ -0,0 +1,36 @@
+const class07Students = new Array();
+
+
+
+//var totalStudents = addStudentToClass("ARNAB");
+//console.log("This variable holds -> " + totalStudents);
+//addStudentToClass("PALLABI");
+
+function addStudentToClass(actualStudentName) {
+    if (actualStudentName == "QueenofDenmark") {
```

Comment:
It should be `queen` to work on codepen. You should test your code at
https://codepen.io/hackyourfuture-cph/pen/PLPmOB